airgun/entities/hostgroup.py

```diff
@@ -0,0 +1,42 @@
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.hostgroup import (
+    HostgroupView,
+    HostgroupEditView,
+)
+
+
+class HostgroupEntity(BaseEntity):
+
+    def set_computeresource(self, hostgroup_name, compute_resource):
```

Comment:
sounds as completely redundant method. Implement basic `update` method instead

Reply:
I will implement the basic update method once there is complete view implemented, together with the rest of CRUD methods. I am planning to do that later, see my longest comment in this PR.

Reply:
Hello @oshtaier . [Contribution info](https://airgun.readthedocs.io/en/latest/#contributing) does not mention anything like this. Or am I missing something?

Reply:
I have changed the method to Update so we do not need to change the interface later.

Reply:
And here i fully agree with @oshtaier - like i said in https://github.com/SatelliteQE/airgun/pull/225#discussion_r235436500, partial implementation is ok, but extra temporary methods which will become useless and redundant with full implementation is definitely not ok.
Thanks @lhellebr for addressing it.